.github/actions/prepare_fixit_venv.yml

```
@@ -36,18 +36,18 @@ runs:
         print(stderr)
         latest_version = search("from versions: (?P<versions>[^)]+)\)", stderr).group("versions").split(", ")[-1]
         print(f"::set-output name=latest_version::{latest_version}")
+    - uses: actions/setup-python@v4
+      id: python-version-setup
+      with:
+        python-version: ${{ inputs.PYTHON_VERSION }}
     - name: download fixit-requirements.txt
       id: fixit_venv_hash
       shell: bash
       run: |
         set -x
         curl -LJO https://raw.githubusercontent.com/carta/.github/main/fixit-requirements.txt
         md5=($(md5sum fixit-requirements.txt))  # the outer parentheses removes the file name
-        echo "::set-output name=venv_hash::fixit-venv-${{ inputs.PYTHON_VERSION }}-$md5-${{ steps.latest_version.outputs.latest_version }}-${{ inputs.FIXIT_LINTER_VERSION }}-v0.1.1"
-    - uses: actions/setup-python@v4
-      id: python-version-setup
-      with:
-        python-version: ${{ inputs.PYTHON_VERSION }}
+        echo "::set-output name=venv_hash::fixit-venv-${{ steps.python-version-setup.outputs.python-version }}-$md5-${{ steps.latest_version.outputs.latest_version }}-${{ inputs.FIXIT_LINTER_VERSION }}-v0.1.1"
```

It should be outputs.version.
```suggestion
        echo "::set-output name=venv_hash::fixit-venv-${{ steps.python-version-setup.outputs.version }}-$md5-${{ steps.latest_version.outputs.latest_version }}-${{ inputs.FIXIT_LINTER_VERSION }}-v0.1.1"
```

---

I think this was renamed, based on the documentation: https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#outputs-and-environment-variables